lib/licensee/project_files/license_file.rb

```diff
@@ -12,8 +12,8 @@ class LicenseFile < Licensee::ProjectFiles::ProjectFile
       # Regex to match any extension except .spdx or .header
       LICENSE_EXT_REGEX = %r{\.(?!spdx|header)[^./]+\z}i.freeze
 
-      # Regex to match any extension except a few unlikely as license
-      # texts with complex filenames
+      # No longer used. Replaced by default IgnoreFile
+      # Maintianed for backward compatability
       OTHER_EXT_REGEX = %r{\.(?!xml|go|gemspec)[^./]+\z}i.freeze
```

Comment:
To answer my own question looking at this, I guess this is needed for backwards compatibility because OTHER_EXT_REGEX is public thus part of the API.